web/client/components/geostory/layouts/sections/Title.jsx

```diff
@@ -56,6 +56,9 @@ export default compose(
     const hideContent = get(focusedContent, "target.id") === contentId;
     const visibility = hideContent ?  'hidden' : 'visible';
     const expandableBackgroundClassName = expandableMedia && background && background.type === 'map' ? ' ms-expandable-background' : '';
+    const overlayStoryTheme = isObject(storyTheme) && isObject(storyTheme.overlay) && storyTheme.overlay || {};
```

Comment:
i think this is sligthly better :)
```suggestion
    const overlayStoryTheme = storyTheme?.overlay || {};
```